packages/electron-updater/src/AppUpdater.ts

```diff
@@ -133,6 +145,14 @@ export abstract class AppUpdater extends (EventEmitter as new () => TypedEmitter
     return this._channel
   }
 
+  get oldBlockMapFileBaseUrl(): string | null {
+    return this._oldBlockMapFileBaseUrl
+  }
+
+  set oldBlockMapFileBaseUrl(value: string | null) {
+    this._oldBlockMapFileBaseUrl = value
+  }
```

Comment:
Are these custom getter/setter needed? There's no data manipulation in either function, so I think we can just have it like this. Having it also on the public property will also have it rendered correctly in the jsdoc.
```
public previousBlockmapBaseUrlOverride: string | null = null
```

Reply:
Fixed